selfdrive/car/honda/values.py

```diff
@@ -1517,6 +1545,7 @@ def init_make(self, CP: car.CarParams):
   CAR.INSIGHT: dbc_dict('honda_insight_ex_2019_can_generated', None),
   CAR.HONDA_E: dbc_dict('acura_rdx_2020_can_generated', None),
   CAR.CIVIC_2022: dbc_dict('honda_civic_ex_2022_can_generated', None),
+  CAR.HRV_2023: dbc_dict('honda_hrv_2023_can_generated', None),
```

Comment:
Where is this DBC? Can you use the radarless Civic one?

Reply:
Probably still not sure. This HR-V is based on Honda Vezel. Not the US model.  Japan/Thailand/Brazil. This data comes from a Brazillian HR-V

Reply:
I will try the civic2022 dbc.